src/models/api-information.model.ts

```diff
@@ -0,0 +1,7 @@
+
+export class ApiInformation {
+  version?: string;
+  // Indexer property to allow additional data
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [prop: string]: any;
```

Comment:
I think we can remove the additional data in the model

Reply:
Additional data removed from model.